packages/react-ui-core/src/Gmap/OverlayView.js

```diff
@@ -111,18 +116,19 @@ export default class OverlayView extends PureComponent {
     let display = 'none'
 
     if (position) {
-      const projection = this.overlay.getProjection()
-
-      // Projection is undefined if map has not yet initialized
-      if (!projection) return
```

Comment:
Why was this removed?

Reply:
Yea good catch. It should not have been removed.